com.sap.cloud.lm.sl.mta/src/test/java/com/sap/cloud/lm/sl/mta/handlers/v3/DescriptorParserTest.java

```diff
@@ -71,6 +71,38 @@
                     new Expectation(Expectation.Type.RESOURCE, "parsed-config-with-partial-schema-version.json"),
                 },
             },
+            // (7) Module hooks with correct data:
+            {
+                "mtad-with-module-hooks-with-correct-data.yaml", null,
+                new Expectation[] {
+                    new Expectation(Expectation.Type.RESOURCE, "parsed-mtad-with-hooks-correct-data.json"),
+                    new Expectation(Expectation.Type.SKIP, null),
+                },
+            },
+            // (8) Module hooks with more than one hook phase:
+            {
+                "mtad-with-module-hooks-with-more-hook-phases.yaml", null,
+                new Expectation[] {
+                    new Expectation(Expectation.Type.RESOURCE, "parsed-mtad-with-hooks-with-more-hook-phases.json"),
+                    new Expectation(Expectation.Type.SKIP, null),
+                },
+            },
+            // (9) Module hooks without type:
+            {
+                "mtad-with-module-hooks-without-type.yaml", null,
+                new Expectation[] {
+                    new Expectation(Expectation.Type.EXCEPTION, "Missing required key \"modules#0#hooks#0#type\""),
+                    new Expectation(Expectation.Type.SKIP, null),
+                },
+            },
+            // (10) Module hooks with two hooks:
+            {
+                "mtad-with-module-hooks-with-two-hooks.yaml", null,
+                new Expectation[] {
+                    new Expectation(Expectation.Type.RESOURCE, "parsed-mtad-with-hooks-with-two-hooks.json"),
```

Comment:
Rename `parsed-mtad-with-hooks-with-two-hooks.json` to `parsed-mtad-with-two-hooks.json`

Reply:
Done